demo/api/src/pages/pages.module.ts

```diff
@@ -7,7 +7,7 @@ import { PagesResolver } from "./pages.resolver";
 
 @Module({
     imports: [MikroOrmModule.forFeature([Page]), forwardRef(() => BlocksModule), RedirectsModule],
-    providers: [PagesResolver, DependenciesResolverFactory.create(Page)],
+    providers: [PagesResolver, DependenciesResolverFactory.create(Page), PagesEntityInfoService],
```

Comment:
This module shouldn't provide a service from the library

Reply:
True